common/src/chain/config/mod.rs

```diff
@@ -145,112 +143,161 @@ pub struct ChainConfig {
 }
 
 impl ChainConfig {
+    /// Bech32m addresses in this chain will use this prefix
+    #[must_use]
     pub fn address_prefix(&self) -> &str {
         &self.address_prefix
     }
 
+    /// The BIP44 coin type for this chain
+    #[must_use]
     pub fn bip44_coin_type(&self) -> ChildNumber {
         self.bip44_coin_type
     }
 
+    /// The genesis block id of the chain
+    #[must_use]
     pub fn genesis_block_id(&self) -> Id<GenBlock> {
         self.genesis_block.get_id().into()
     }
 
+    /// The genesis block of the chain
+    #[must_use]
     pub fn genesis_block(&self) -> &Arc<WithId<Genesis>> {
         &self.genesis_block
     }
 
+    /// The bytes that are used to prefix p2p communication to uniquely identify this chain
+    #[must_use]
     pub fn magic_bytes(&self) -> &[u8; 4] {
         &self.magic_bytes
     }
 
+    /// The port that the p2p server will listen on
+    #[must_use]
     pub fn p2p_port(&self) -> u16 {
         self.p2p_port
     }
 
+    /// The current version of the protocol that this chain is running
+    #[must_use]
     pub fn version(&self) -> &SemVer {
         &self.version
     }
 
+    /// The chain of this config (mainnet, testnet, regtest, etc...)
+    #[must_use]
     pub fn chain_type(&self) -> &ChainType {
         &self.chain_type
     }
 
+    /// The mechanism by which we define changes in the chain, including consensus and other upgrades/forks
+    #[must_use]
     pub fn net_upgrade(&self) -> &NetUpgrades<UpgradeVersion> {
         &self.net_upgrades
     }
 
+    /// Checkpoints enforced by the chain, as in, a block with vs height that must be satisfied
```

Comment:
width

Reply:
I misunderstood what you meant. I'll fix this in the next PR.